stdlib/public/runtime/Metadata.cpp

```diff
@@ -4207,6 +4301,110 @@ static bool doesNotRequireInstantiation(
   return true;
 }
 
+static const unsigned swift_ptrauth_key_associated_type =
```

Comment:
Was it intentional to assign this variable to itself? Clang accepts this without warning :/